backend/source/mutations/DeleteSourceMutation.py

```diff
@@ -0,0 +1,21 @@
+from graphene import Mutation, ID, Boolean, ResolveInfo, String
+from source.models import Source
+
+
+class DeleteSourceMutation(Mutation):
```

Comment:
The implementation of this looks fine, but deleting sources is quite destructive, so it may be better to restrict this to the admin interface.